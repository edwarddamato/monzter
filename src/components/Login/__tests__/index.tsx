import * as React from 'react';
import { Login } from '../';
import { create } from 'react-test-renderer';

it('renders correctly', () => {
  const login = create(
    <Login loginUrl='http://someurl' />
  ).toJSON();
  expect(login).toMatchSnapshot();
});