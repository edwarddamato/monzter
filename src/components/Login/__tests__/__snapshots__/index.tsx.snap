// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<section
  className="component_login"
>
  <img
    className="component_login-logo"
    src="test-file-stub"
  />
  <a
    href="http://someurl"
  >
    Login to Monzo
  </a>
</section>
`;
